src/YoutubeDl.php

```diff
@@ -111,7 +117,11 @@ public function onProgress(callable $onProgress)
 
     public function download(string $url): Video
     {
-        if (!$this->downloadPath) {
+        if(isset($this->options['output'])) {
```

Comment:
What if people uses output option only as the file name template?

Reply:
Hello,

You are right, my fault.. this use case was forgotten.

I made some modifications for simplify the code, no extra parameters is needed in the constructor class, the combinations are the next:

Output no defined + Download path defined -> uses download path
Output defined with full path + down load path defined -> uses output path
Output defined as filename template only + download path defined -> use download path
No output + no download path -> return error no path defined